### DISCUSSION TITLE
/articles/sharepoint-online-managing-site-collection-app-catalogs-with-the-office-365-cli/

### DISCUSSION THREAD

**Discussion by vtns-comment-importer** [Voitanos/voitanos.io-comments#179] [Comments]
Managing SharePoint Online Site Collection App Catalogs with the Office 365 CLI
https://www.andrewconnell.com/articles/sharepoint-online-managing-site-collection-app-catalogs-with-the-office-365-cli/

**Comment by vtns-comment-importer** [comment]
```header
Originally posted by gene on 2018-04-26 12:09
```

Just an FYI - adding an app catalog doesn't work if you're executing as an account having tenant granted SharePoint Admin rights. Vesa Juvonen suggests this is "by design".

**Comment by Adam-it** [comment]
hey AC 👋
I noticed one tiny detail in the code snippets you have on this page.

In the `m365 spo site appcatalog remove` and `m365 spo site appcatalog add` you are using an `url` option which was renamed to `siteUrl` in the latest version of CLI for Microsoft 365. May I kindly ask you to perform a fixup 👍

You may refer to the docs for more info about that: https://pnp.github.io/cli-microsoft365/cmd/spo/site/site-appcatalog-add

Thanks a ton
You Rock 🤩

**Reply by andrewconnell** [reply]
Thanks, good catch! Updated and pushing the changes now.
